src/creature.js

```diff
@@ -1695,76 +1694,6 @@ export class Creature {
 		);
 	}
 
-	getBuffDebuff(stat) {
```

Comment:
None of this code was being used (assume it's related to the UI you said broke a while ago), so I've removed it to avoid confusion.